ts/features/walletV3/details/components/WalletDetailsPaymentMethodScreen.tsx

```diff
@@ -0,0 +1,173 @@
+import FocusAwareStatusBar from "../../../../components/ui/FocusAwareStatusBar";
+import { WalletInfo } from "../../../../../definitions/pagopa/walletv3/WalletInfo";
+import { isDesignSystemEnabledSelector } from "../../../../store/reducers/persistedPreferences";
+import { walletDetailsInstrumentPotSelector } from "../store";
+import { walletDetailsDeleteInstrument } from "../store/actions";
+
+type Props = {
+  paymentMethod?: WalletInfo;
+  card: React.ReactNode;
+  content: React.ReactNode;
+  headerTitle?: string;
+};
+
+// ----------------------------- component -----------------------------------
+
+/**
+ * Base layout for payment methods screen & legacy delete handling
+ */
+const WalletDetailsPaymentMethodScreen = (props: Props) => {
+  const { card, content, paymentMethod } = props;
+  const hasErrorDelete = pot.isError(
+    useIOSelector(walletDetailsInstrumentPotSelector)
+  );
+  const [isLoadingDelete, setIsLoadingDelete] = React.useState(false);
+  const dispatch = useIODispatch();
+  const isDSenabled = useIOSelector(isDesignSystemEnabledSelector);
+  const blueHeaderColor = isDSenabled ? IOColors["blueIO-600"] : IOColors.blue;
+
+  const navigation = useNavigation();
+
+  const deleteWallet = (walletId: string) =>
+    dispatch(
+      walletDetailsDeleteInstrument.request({
+        walletId,
+        onSuccess: _ => {
+          IOToast.success(I18n.t("wallet.delete.successful"));
+          navigation.goBack();
+        },
+        onFailure: _ => {
+          IOToast.error(I18n.t("wallet.delete.failed"));
+        }
+      })
+    );
+
+  React.useEffect(() => {
+    if (hasErrorDelete) {
+      setIsLoadingDelete(false);
+    }
+  }, [hasErrorDelete]);
+
+  const onDeleteMethod = () => {
+    // Create a native Alert to confirm or cancel the delete action
+    Alert.alert(
+      I18n.t("wallet.newRemove.title"),
+      I18n.t("wallet.newRemove.body"),
+      [
+        {
+          text:
+            Platform.OS === "ios"
+              ? I18n.t(`wallet.delete.ios.confirm`)
+              : I18n.t(`wallet.delete.android.confirm`),
+          style: "destructive",
+          onPress: () => {
+            if (paymentMethod) {
+              deleteWallet(paymentMethod.walletId);
+            }
+          }
+        },
+        {
+          text: I18n.t("global.buttons.cancel"),
+          style: "default"
+        }
+      ],
+      { cancelable: false }
+    );
+  };
+
+  if (isLoadingDelete) {
+    return (
+      <LoadingSpinnerOverlay
+        isLoading={true}
+        loadingCaption={I18n.t("cardComponent.deleteLoading")}
+      />
+    );
+  }
+
+  return (
+    <BaseScreenComponent
+      contextualHelp={emptyContextualHelp}
+      headerTitle={props.headerTitle ?? ""}
+      faqCategories={["wallet_methods"]}
+      goBack={true}
+      titleColor="white"
+      dark={true}
+      headerBackgroundColor={blueHeaderColor}
+    >
+      <FocusAwareStatusBar barStyle="light-content" />
```

Comment:
```suggestion
      <StatusBar barStyle="light-content" />
```
After some testing, I noticed that this screen renders the statusBar as black even though you have set it to light, so
I've tried moving this component around the screens, but this seems to be the only fix; if you find another way to have the same result, it will probably be a better way.

Reply:
Good catch! Addressed into 24ac6f398b8d8120a3f8947b2f6794c6d9850a4d